### ISSUE TITLE
Libs were not updated in npm for version 2.2.2

### ISSUE THREAD

**Issue by jharris4** [th0r/webpack-bundle-analyzer#46]
#### Issue description

```
cd ~/temp
npm pack webpack-bundle-analyzer@2.2.2
tar -zxvf webpack-bundle-analyzer-2.2.2.tgz
```

```
ls -al package/src/parseUtils.js
6372  8 Feb 15:42 package/src/parseUtils.js
```

```
ls -al package/lib/parseUtils.js
4228  6 Jan 04:25 package/lib/parseUtils.js
```

Looks like the lib directory didn't get updated when 2.2.2 was published to npm.

#### Technical info
* Webpack Bundle Analyzer version: 2.2.2
* Webpack version: 2.2.1
* Node.js version: 7.5
* npm/yarn version: 4.1.2
* OS: MacOS 10.12.3

#### Debug info
How do you use this module? As CLI utility or as plugin?

If CLI, what command was used? (e.g. `webpack-bundle-analyzer -O path/to/stats.json`)

If plugin, what options were provided? (e.g. `new BundleAnalyzerPlugin({ analyzerMode: 'disabled', generateStatsFile: true })`)

What other Webpack plugins were used?

It would be nice to also attach webpack stats file.
It can be generated using these options:
```js
new BundleAnalyzerPlugin({
  analyzerMode: 'disabled',
  generateStatsFile: true,
  // Excludes module sources from stats file so there won't be any sensitive data
  statsOptions: { source: false }
})`
```
`stats.json` will be created in Webpack bundle output directory.

**Comment by th0r** [comment]
Sorry about that, my mistake 😬

**Comment by th0r** [comment]
Fixed in `v2.2.3`
